qubes/tests/__init__.py

```diff
@@ -1127,6 +1127,50 @@ def list_templates():
             _templates = ()
     return _templates
 
+def create_testcases_for_templates(name, *bases, globals, **kwds):
```

Comment:
If `globals` argument is needed (see below), I'd name it differently, to not cover `globals` builtin.

Reply:
It is named such for the function's signature, so the caller has to name it globals. It makes error more readable:
```
TypeError: create_testcases_for_templates() missing 1 required keyword-only argument: 'globals'
```
If you'd like, you can `import builtins` or even `from builtins import globals as _globals` or whatever.